config/cypress.config.json

```diff
@@ -5,5 +5,7 @@
   "supportFile": "tests/e2e/support/index.js",
   "fixturesFolder": "tests/e2e/fixtures",
   "videosFolder": "tests/e2e/videos",
-  "video": false
+  "video": false,
+  "viewportWidth": 1440,
```

Comment:
Viewport height and width can be removed now since the app no longer uses viewport to determine device.

Reply:
Done.